Paybot template

fraolb opened the pull request:
This PR adds a new Celo Paybot Frame to the celo-farcaster-frames repo. The frame allows users to search for Farcaster profiles and send Celo tokens as tips. It includes:

- Demonstrate how to use Celo network on Farcaster Frame

- A Farcaster-integrated search form

- Wallet connection using RainbowKit & Wagmi

- Transaction processing with Celo blockchain

resolves issue #3

GigaHierz commented:
Hi Fraol, this looks great. 

could you please:

1. move the celo-paybot folder out of the template folder? In the end you should just add one project on root level.
2. add a link to a live deployment so we can test the frame

fraolb commented:
Hello @GigaHierz I have moved the celo-paybot folder out of the template folder and also, I have added the live deployed frame in the readme file (https://farcaster-frame-celo-template.vercel.app/)
